web/source/osk/anchoredOskView.ts

```diff
@@ -59,12 +61,34 @@ namespace com.keyman.osk {
       s.position = 'fixed';
     }
 
-    protected postKeyboardLoad() {
-      // Initializes the size of a touch keyboard.
+    /**
+     * @override
+     */
+    public refreshLayout(pending?: boolean): void {
+      // Prevent a circular call-stack chain.
```

Comment:
Can we explain in this comment how we'd get a recursion -- I think that what `isResizing` is doing is clear enough, but tracing through to figure out how we get recursion and why that is not a bug would be helpful to me.

Reply:
![image](https://user-images.githubusercontent.com/25213402/174508231-2488e11f-8f45-4175-a219-0e34bea5d1e9.png)

https://github.com/keymanapp/keyman/blob/c66bfac43f63c829d6b7d226029f26951cd49021/web/source/osk/anchoredOskView.ts#L82-L87

https://github.com/keymanapp/keyman/blob/c66bfac43f63c829d6b7d226029f26951cd49021/web/source/osk/anchoredOskView.ts#L138-L145

For the line 441 call...

https://github.com/keymanapp/keyman/blob/c66bfac43f63c829d6b7d226029f26951cd49021/web/source/osk/oskView.ts#L404-L404

https://github.com/keymanapp/keyman/blob/c66bfac43f63c829d6b7d226029f26951cd49021/web/source/osk/oskView.ts#L441-L442

And you can probably follow it from there, given the nomenclature.

Reply:
Note that `setSize` can also be used on OSK init and is called by different subclasses of OSKView.

For example, the desktop-style `FloatingOSKView` calls it when the user manually resizes the OSK during a click & drag on the resize handle.

Reply:
Thanks. Could you summarize in the code comment?